### PR TITLE
Rename release channels to preview, stable and vs

### PR DESCRIPTION
**Pull request by luhenry**

### CODE REVIEW

**Comment by directhex** [comment]
```
  *  linking to /download/alpha#download-lin, but download-lin does not exist (line 107)
     <a href="/download/alpha#download-lin">alpha</a>
```

I don't know what Travis is talking about. The anchors haven't changed, and the `redirect_from`'s seem fine too? @akoeplinger?

File: download/stable.html [hunk 1]
```diff
@@ -1,17 +1,23 @@
...
+title: Download - Stable
 navgroup: download
 redirect_from:
+  - /download/
   - /downloads/
   - /Downloads/
-  - /Download/
+  - /beta/
+  - /Beta/
+  - /download/beta/
```
Comment (akoeplinger) [inline download/stable.html]: beta stuff should be moved to preview.html

File: download/vs.html [hunk 2]
```diff
@@ -1,16 +1,18 @@
...
 ---
+
 <div class="row">
   <div class="small-12 columns">
-    <h1>Beta Download</h1>
+    <h1>Download</h1>
+
+    <p><b>Release channels:</b>&nbsp;<span><a href="/download/preview/">Preview</a> - <a href="/download/stable/">Stable</a> - <a href="/download/vs/">VS</a></span></p>
+
+    <p style="padding: 0 20px"><b>In the VS release channel are packages which are shipped as part of a Visual Studio for Mac release, they are the most stable but also the least frequently updated packages.</b></p>
```
Comment (akoeplinger) [inline download/vs.html]: `In the VS release channel are packages` -> `The VS release channel contains packages`

File: _posts/2017-05-15-mono-5-0-is-out.md [hunk 3]
```diff
@@ -5,7 +5,7 @@ author: Alexander Köplinger
 tags: [releases]
 ---
 
-Mono 5.0 is out in the [stable channel](/download/) !
+Mono 5.0 is out in the [stable channel](/download/stable) !
```
Comment (akoeplinger) [inline _posts/2017-05-15-mono-5-0-is-out.md]: please add a trailing `/` to all urls to save a redirect

File: download/stable.html [hunk 4]
```diff
@@ -1,17 +1,23 @@
 ---
 layout: base
-title: Download
+title: Download - Stable
 navgroup: download
 redirect_from:
+  - /download/
   - /downloads/
   - /Downloads/
-  - /Download/
```
Comment (akoeplinger) [inline download/stable.html]: this should be kept

**Review by akoeplinger — CHANGES_REQUESTED** [review]
few small comments, otherwise looks good!

**Review by akoeplinger — APPROVED** [review]

**Pull request merged**